backend/src/core/loaders/sequelize.loader.ts

```diff
@@ -20,17 +21,25 @@ import { SmsMessage, SmsTemplate, SmsOp } from '@sms/models'
 import logger from '@core/logger'
 
 const DB_URI = config.get('database.databaseUri')
+const DB_READ_REPLICA_URI = config.get('database.databaseReadReplicaUri')
 
 const sequelizeLoader = async (): Promise<void> => {
   const dialectOptions = config.get('IS_PROD')
     ? config.get('database.dialectOptions')
     : {}
-  const sequelize = new Sequelize(DB_URI, {
+  const sequelize = new Sequelize({
     dialect: 'postgres',
     logging: false,
     pool: config.get('database.poolOptions'),
+    replication: {
+      read: [parse(DB_READ_REPLICA_URI)],
```

Comment:
I believe you need to set "username" for this. I couldn't get it to work on staging. ie parse should return `{auth, username, host, port, database}` instead of `{auth, user, host, port, database}`